text/0000-nested-publish.md

```diff
@@ -131,6 +131,8 @@ When a nested dependency is present (making its referent be a nested package), t
 
   This check is intended only to prevent accidents (such as vendoring a third-party package without considering the implications of redistributing it). It is always valid to omit `package.license` from the nested package, thus making no machine-readable claims about its licensing.
 
+It is an error for any two of the packages in the transitive closure of nested dependencies (including the parent package) to share a package name. This is validated by all Cargo operations that would generate or read a lockfile.
```

Comment:
Please rewrite without talking about dependency closures.  That is fairly jargony and jargon can get in the way of communication.  For example, I've been working on multiple build systems for over a decade and yet I have no idea what the use of "closure" means in this context.

Reply:
Done.